lib/optimize/ModuleConcatenationPlugin.js

```diff
@@ -64,6 +64,12 @@ class ModuleConcatenationPlugin {
 						continue;
```

Comment:
Could you change `parser.state.module.buildMeta.hasEval = true` to `parser.state.module.buildInfo.moduleConcatenationBailout = "eval()"`